### DISCUSSION TITLE
(More of a query)With regards to Mod workshop folder behaviour

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#3090] [Q&A] [answered]
I noticed a few updates back, the mod folder 304930 which used to keep all the workshop files that were used by any servers that we played in, started self deleting some of the mods within that weren't manually subscribed to.

Not that an automated cleanup is bad per say, but it gets really annoying recently when I do have to re download the mods every other time when entering the same server simply due to this automated deletion. (not to mention some missing assets errors that pop up time to time, maybe due to my slow internet.)

So erm the question:
Is this an unturned setting that I missed somewhere along the updates?
Or is this a new steam client thingy from the steam client updates that does this workshop deleting?
(Or maybe it deletes itself randomly everytime I update the steam client/logout....I don't really know for now)

**Comment by SDGNelson** [comment, marked as answer]
Sorry, this would be Steam. Unturned does not manually delete any unused workshop files.

**Reply by Ongjming** [reply]
Thanks! :)
